docs/faststore/docs/headless-cms/multiple-page-template.mdx

```diff
@@ -0,0 +1,151 @@
+
+#### Rendering within a `department`
+
+Products in [catalog](https://help.vtex.com/en/tutorial/catalog-overview--77M8ItLhDXs6aBdQTqToVe) fall under a [category tree](https://help.vtex.com/en/tutorial/what-is-a-category--6HV4Q3E2FauUoOQoiCCgCg), typically `/department/category/subcategory/`. For example, the product we’ve been using is in the `Technology department`, allowing you to create a department-specific template.
+
+![PDP Department template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpdepartment___387cf641f675d05666afeafc65a3e56b.png)
+
+If this product is under a [subcategory](https://help.vtex.com/en/tutorial/what-is-a-subcategory--2cb0aRkG3i6AeiAMM24iwY?&utm_source=autocomplete), you can create a corresponding template. The category tree is used in the template input, separated by slashes.
+
+![PDP subcategory template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpsubcategory___566be1fd3bba2db777dd18881933292c.png)
+
+You can also see the category hierarchy through the breadcrumb.
+
+![PDP breadcrumb](https://vtexhelp.vtexassets.com/assets/docs/src/breadcrumb___c9c12da51f685572dfb3a9403168b26a.png)
+
+#### Rendering the generic Product Page template
+If no matches are found for the slug or department, the generic template will be used to render the PDP. In this case, create a Headless CMS document with the `Product Page` option, leaving the **Template** field empty in the **Sections** tab
+
+![PDP general template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpgeneral___cdeeaf6060b529aea029e2e5814efc50.png)
+
+### Product Listing Page (PLP) template selection example
+
+For PLPs, we will go over the following possible templates:
+- [Slug](#rendering-within-slugs)
+- [Product Listing Page generic template](#rendering-the-generic-product-listing-page).
+
+When using PLPs, this type of page template is recognized by the slug pattern. In the example, we will use the [office/chairs slug](https://starter.vtex.app/office/chairs) to render the PLP template.
+
+#### Rendering within slugs
+
+Products in catalog are organized within a category tree, structured as `/department/category/subcategory`. If a slug template exists for any of these options, it will be rendered accordingly.
+
+For example, if you want to render a specific template for products inside a [department](https://help.vtex.com/en/tutorial/what-is-a-department--22rKjmYWVmmKAK8CWa8yKw?&utm_source=autocomplete) without considering the category or subcategory, you can create a Department template [/office](https://starter.vtex.app/office/chairs).
+
+![PLP department template](https://vtexhelp.vtexassets.com/assets/docs/src/plpdepartment___4c520237721ca14a722215a93e8cd4c7.png)
+
+
+If this product were under a `subcategory`, it would still be possible to create a template representing this scenario. The category tree is used in the template slug input, as in the image below.
```

Comment:
```suggestion
If this product were under a `subcategory`, it would still be possible to create a template representing this scenario. The category tree is used in the template slug input, as in the image below.
```